src/main/java/org/terasology/biomesAPI/Biome.java

```diff
@@ -57,4 +57,12 @@ default short biomeHash() {
 
         return hash;
     }
+
+    default float getHumidity() {
+        return .3f;
+    }
+
+    default float getTemperature() {
```

Comment:
This definitely needs JavaDoc! What is the scale of temperature here (e.g., what are possible values)? How should the be interpreted?

Also, I'm wondering whether this is it rather a _base temperature_ of sorts? If the biome fully defines the temperature, there's no need for additional systems, is there?

Reply:
Is there a reason not to encode `baseTemeperature` and `baseHumidity` also in the method name? I'd like to avoid confusion where a module developer does not know where to get the temperature of a biome or block from. `biome.getTemperature()` looks like it would do the right thing, whereas `biome.getBaseTemperature()` directly yields the information that the _actual temperature_ might be different...